crates/re_viewer/data/quick_start_guides/cpp_connect.md

```diff
@@ -1,3 +1,45 @@
 ## C++ Quick Start
 
-TODO(ab): https://github.com/rerun-io/rerun/issues/3870
+${SAFARI_WARNING}
+
+#### Installing the Rerun viewer
+The Rerun C++ SDK works by connecting to an awaiting Rerun Viewer over TCP.
+
+Building from source:
+* [Install cargo](https://rustup.rs/)
+* `cargo install rerun-cli`
+
+Using `pip`:
+* `pip install rerun-sdk`
+
+After you've installed it, type `rerun` in your terminal to start the viewer.
+
+
+#### Using the Rerun C++ SDK
+First install the `arrow-cpp` library on your system using your favorite package manager.
```

Comment:
I'd suggest we add something like

```
Find more information at the official Arrow Apache [install guide](https://arrow.apache.org/install/).
```

I'm also leaning towards mentioning `pixi global install arrow-cpp` because that seems to be one of the easiest ways and I think we should start advertising it. But I don't know yet how to formulate this in the least awkward way.